documents/course-delivery.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: page
+title: Help
+subtitle: Teaching Strategies and Course Delivery
+---
+
+### Flipped Classroom
+
+I use a flipped, learner-centered, approach to teach this course, because I 
+think that learning to program has very little to do with me talking and a lot 
+to do with students working on computers. Flipped classes work well for all 
+kinds of content, but I think they work particularly well for computer oriented 
+classes. If you're interested in knowing more take a look at this great
+info-graphic:
```

Comment:
We should remove the use of "I" from this paragraph since it's now a group effort:

"This course uses a flipped... because learning to program is most effective when students are actively working with computers... but they work particularly..."